package.json

```diff
@@ -11,30 +11,28 @@
   "bugs": {
     "url": "https://github.com/cypress-io/cypress-realworld-app/issues"
   },
-  "engines": {
-    "node": "^16.16.0"
-  },
```

Comment:
I did a smoke test on Node.js 14, 16, 18 & 20.
yarn fails on 14 and works on >=16
yarn dev works on >=16

It seems that it is not essential to have `engines` defined. I'm wondering though if it would be safer to add it back in, notwithstanding that `.node-version` & `.nvmrc` are at the `18.16.x` level.

If you want to avoid RWA being used on `16.x`, you could exclude it. 

Depending on intentions, this might be a good definition:

```
  "engines": {
    "node": "^16.16.0 || ^18.0.0 || >=20.0.0"
```


```text
yarn install v1.22.19
[1/4] Resolving packages...
warning Resolution field "@types/express@4.17.2" is incompatible with requested version "@types/express@^4.17.14"
[2/4] Fetching packages...
error jsdom@22.1.0: The engine "node" is incompatible with this module. Expected version ">=16". Got "14.21.3"
error Found incompatible module.

```

Reply:
ahh, ya i didn't mean to remove that permanently, I'll add your suggestion.